src/components/ont/runs/ONTRunInformation.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <traction-section number="1" title="Run Information">
+    <div class="flex flex-row">
+      <div class="flex flex-col w-1/2">
+        <traction-field-group label="Instrument" label-for="instrument-selection">
+          <traction-select
+            id="instrument-selection"
+            :options="instrumentOptions"
+            :value="instrumentName"
+            :disabled="!newRecord"
+            @input="setInstrumentName"
```

Comment:
Up to you but you could set the whole instrument here. e.g. calls a mutation which sets this.currentRun.instrument to the instrument based on the name. If you then needed the instrument name in other areas you could use this.currentRun.instrument.name.

Reply:
Hi yes I did way up about 4 different ways of doing this. And went for the approach to get the whole instrument via the `getter`. Which still provided some nice refactoring.

Reply:
OK cool, makes sense. Yeah the instrument refactoring you did looks good.